src/sequence.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  FindRoute,
+  inject,
+  InvokeMethod,
+  parseOperationArgs,
+  ParsedRequest,
+  Reject,
+  Send,
+  ServerResponse,
+  SequenceHandler,
+} from '@loopback/core';
+
+import {
+  AuthenticateFn,
+} from '@loopback/authentication';
+
+export class MySequence implements SequenceHandler {
+  constructor(
+    @inject('sequence.actions.findRoute') protected findRoute: FindRoute,
+    @inject('sequence.actions.invokeMethod') protected invoke: InvokeMethod,
+    @inject('sequence.actions.send') protected send: Send,
+    @inject('sequence.actions.reject') protected reject: Reject,
+    @inject('authentication.actions.authenticate')
+    protected authenticateRequest: AuthenticateFn,
+  ) {}
+
+  async handle(req: ParsedRequest, res: ServerResponse) {
+    try {
+      const route = this.findRoute(req);
+
+      fakeAuthenticate(req);
+
+      const user = await this.authenticateRequest(req);
+
+      const args = await parseOperationArgs(req, route);
```

Comment:
Please inject this via `sequence.actions.parseParams`, see

https://github.com/strongloop/loopback-next/blob/54cbd59b4b439d613d1530ea09751267c7b6ca94/packages/core/src/sequence.ts#L79

and

https://github.com/strongloop/loopback-next/blob/54cbd59b4b439d613d1530ea09751267c7b6ca94/packages/core/src/sequence.ts#L105